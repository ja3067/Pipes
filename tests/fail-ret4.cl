if True:
    def foo(x) -> float:
        return x

x = 3.0
print(foo(x))

x = "hello"
print(foo(x))
