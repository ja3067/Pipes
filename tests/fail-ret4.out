3
RuntimeError: invalid return type (expected float)
